enterprise/app/history/history.tsx

```diff
@@ -415,15 +429,15 @@ export default class HistoryComponent extends React.Component<Props, State> {
                   <span>
                     <span className="history-title">{this.props.username}'s builds</span>
                     <a className="history-button" href={`/trends/?user=${this.props.username}`}>
-                      <BarChart2 /> Trends
+                      <BarChart2 /> View Trends
```

Comment:
nit: do we want to make these `Sentence case` like we do for other labels?

Reply:
I went back and forth on this one, but ok - went with sentence case